xbmc/addons/interfaces/Addon/AddonCallbacksAddon.h

```diff
@@ -80,6 +80,7 @@ class CAddonCallbacksAddon
   static bool FileExists(const void* addonData, const char *strFileName, bool bUseCache);
   static int StatFile(const void* addonData, const char *strFileName, struct __stat64* buffer);
   static char *GetFileProperty(const void* addonData, void* file, XFILE::FileProperty type, const char *name);
```

Comment:
For consistency reasons, I would rename all existing single name - single value methods to `GetFilePropertyValue`.